packages/core/src/Table/stories/TableHooks/UseHvRowState.tsx

```diff
@@ -0,0 +1,123 @@
+import { useMemo } from "react";
+import {
+  HvTable,
+  HvTableBody,
+  HvTableContainer,
+  HvTableHead,
+  HvTableHeader,
+  HvTableRow,
+  HvTableCell,
+  useHvData,
+  useHvRowState,
+  HvInput,
+  HvTableColumnConfig,
+  HvIconButton,
+} from "@hitachivantara/uikit-react-core";
+
+import { Close, Edit } from "@hitachivantara/uikit-react-icons";
+
+import { makeData, AssetEvent } from "../storiesUtils";
+
+const EditableCell = ({ value }) => <HvInput defaultValue={value} />;
```

Comment:
Let's hoist the toggle to this component (reasons above). Also, we should type the arguments (`noImplicitAny`)

```suggestion
const EditableCell = ({ value, row }: HvCellProps<AssetEvent, string>) => {
  return row.state?.isEditing ? <HvInput value={value} /> : value;
};
```